### DISCUSSION TITLE
Swipe function for mobile devices

### DISCUSSION THREAD

**Discussion by starmomou** [Budibase/budibase#3341] [Ideas]
I think it would be nice to have a feature to swipe (left/right) between pages(or images) while using tablet.

**Comment by mjashanks** [comment]
Thanks for this. Would it be possible to give me a bit more detail on your use case / what you are trying to build?

I'm not sure what swiping between pages would do? 
Would this move to the "Next" record? If so, how would we determine what the "Next" record is?
Is supposed to change the URL? Again, how do we determine what the next URL is?

If it's swiping between images (attachments) on the same screen then that's different, probably easier.

Thanks!

**Reply by starmomou** [reply]
Thanks for your reply! There are too many use cases I can think of with the swipe function but now my intention is to make the app more tablet-friendly because sometimes it is difficult for some people to access internal system with their computers.
Also, I think a **slide navigation bar** for hiding navigation bar would be great, especially for the side nav :D

This is just a direction for the swipe feature.
<img width="1133" alt="螢幕截圖 2021-11-14 上午1 21 01" src="https://user-images.githubusercontent.com/73826858/141654268-30d44a72-c7de-4c55-9035-9748dbcd7c10.png">

**Reply by mjashanks** [reply]
Ahh "page" as in "pagination", sorry didn't pick up on that!

Makes sense!
